modules/nw-ovn-kubernetes-live-migration-about.adoc

```diff
@@ -88,6 +88,8 @@ During the limited live migration, both OVN-Kubernetes and OpenShift SDN run in
 
 * OVN-Kubernetes reserves the `100.64.0.0/16` and `100.88.0.0/16` IP address ranges. These subnets cannot be overlapped with any other internal or external network. If these IP addresses have been used by OpenShift SDN or any external networks that might communicate with this cluster, you must patch them to use a different IP address range before starting the limited live migration. See "Patching OVN-Kubernetes address ranges" for more information.
 
+* If your cluster uses the OVN-Kubernetes network plugin and PTP uses the User Datagram Protocol (UDP) for hardware time stamping, the hardware time stamping cannot be applied to primary interface devices, such as an Open vSwitch (OVS) bridge.
```

Comment:
I assumed users is using openshfit-sdn plugin when reading this part.  and want to migration to ovn-kubernetes cluster.  so can we update with
```
if your openshift-sdn cluster with PTP uses the User Datagram Protocol (UDP) for hardware time stamping,  Please note, after migration to ovn-kubernetes. the hardware time stamping cannot be applied to primary interface devices, such as an Open vSwitch (OVS) bridge. So configurations like UDPv4 cannot work with the `br-ex` interface.
```

Reply:
Thanks, updated!